sklearn/impute/_iterative.py

```diff
@@ -100,6 +100,13 @@ class IterativeImputer(BaseEstimator, TransformerMixin):
         "random"
             A random order for each round.
 
+    skip_non_missing_features : boolean, optional (default=False)
```

Comment:
Maybe we should have:
```
    skip : 'none' (default), 'complete'
        Which features to use the initial imputation for, instead of iterative imputation.
        'complete' avoids learning an iterative imputation model for features that
        have no missing values in fitting, so will be efficient if you expect those features
        not require imputation. 
```
We could consider supporting a list of feature indices too.

Reply:
I like boolean a little better because it's easy to remember what the options are. People could easily confuse `'none'` for `None`. 

As for supporting feature indices: I can't really think of when I'd want to do this and how exactly it would work. Let's keep it simple for now.